sycl/doc/extensions/experimental/sycl_ext_oneapi_properties.asciidoc

```diff
@@ -169,76 +170,50 @@ namespace sycl::ext::oneapi::experimental {
 template<typename...> struct property_value;
 
 // This property has no parameters.
-struct bar_key {
-  using value_t = property_value<bar_key>;
-};
-// bar is an object of the property value type of bar.
-inline constexpr bar_key::value_t bar;
+inline constexpr property_value<struct bar_key> bar;
 
 // This property has one integer non-type parameter.
-struct baz_key {
-  template<int K>
-  using value_t = property_value<baz_key, std::integral_constant<int, K> >;
-  // Note: integral_constant is used for the example. An implementation can use a different mapping.
-};
-// baz is an object of a property value type of baz.
 template<int K>
-inline constexpr baz_key::value_t<K> baz;
+inline constexpr property_value<struct baz_key, std::integral_constant<int, K>> baz;
 
 // This property has an arbitrary number of type parameters.
-struct boo_key {
-  template<typename...Ts>
-  using value_t = property_value<boo_key, Ts...>;
-};
-// boo is an object of a property value type of boo.
 template<typename... Ts>
-inline constexpr boo_key::value_t<Ts...> boo;
+inline constexpr property_value<struct boo_key, Ts...> boo;
 
 } // namespace experimental::oneapi::ext::sycl
 ```
 
 === Property traits
 
-All runtime and compile-time-constant properties must have a specialization of 
-`is_property_key` and `is_property_value` that inherits from
-`std::true_type`, and they must have a specialization of `is_property_key_of`
+All properties must have a specialization of `is_property_key_of`
```

Comment:
I think this should not be required.  I've defined a number of properties already that do not have a specialization of `is_property_key_of`.